cloudify_agent/tests/api/pm/__init__.py

```diff
@@ -14,13 +14,14 @@
 #  * limitations under the License.
 
 import os
-import nose.tools
+import unittest
 import inspect
 import types
 from functools import wraps
 from mock import _get_target
 from mock import patch
 
+
```

Comment:
no need in 2 empty lines.
also, need empty line after line 19.
sort imports by line length.